scripts/change_file_ext.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
```

Comment:
Rename to change_file_extensions.py -- don't use abbreviations if possible.

Reply:
Done.